tests/apps/detection/networks/test_retinanet.py

```
@@ -136,18 +136,18 @@ def test_retina_shape(self, model, input_param, input_shape):
             self.assertEqual(tuple(cc.shape for cc in result[net.cls_key]), expected_cls_shape)
             self.assertEqual(tuple(cc.shape for cc in result[net.box_reg_key]), expected_box_shape)
 
-    @parameterized.expand(TEST_CASES_TS)
-    def test_script(self, model, input_param, input_shape):
+    @parameterized.expand(TEST_CASES_EXPORT)
+    def test_export(self, model, input_param, input_shape):
         try:
-            idx = int(self.id().split("test_script_")[-1])
+            idx = int(self.id().split("test_export_")[-1])
         except BaseException:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `BaseException`

(BLE001)